src/styles/global/storybook-components.scss

```diff
@@ -55,4 +55,5 @@
 @import "src/components/ui/SlideOver/slide-over.scss";
 @import "src/components/ui/SROnly/sr-only.scss";
 @import "src/components/ui/Tiles/tiles.scss";
+@import "src/components/ui/Toast/toast.scss";
```

Comment:
If we also import it in `pages/layout.scss`, will it work for the app?

![CleanShot 2022-05-16 at 19 51 53](https://user-images.githubusercontent.com/381395/168695052-91bc040d-1895-4d92-9fce-cac3578bd64e.png)